docs/Bootloaders.md

```diff
@@ -553,8 +558,8 @@ Begin by downloading and compiling the software (each step may take
 several minutes and the "make" step may take 30+ minutes):
 
 ```
-sudo apt-get update
-sudo apt-get install autoconf libtool telnet
+sudo apt update
+sudo apt install autoconf libtool telnet
```

Comment:
I'm not sure about the implications of making this change.  As far as I know, `apt-get` is always installed, but I'm not sure `apt` is always installed.